packages/core/src/components/slider/sliderTrackStop.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+import { Intent } from "../../common";
+
+export interface ISliderTrackStopProps {
+    value: number;
+    trackIntentAfter?: Intent;
+    trackIntentBefore?: Intent;
+}
+
+export class SliderTrackStop extends React.PureComponent<ISliderTrackStopProps> {
```

Comment:
i feel like we could easily fold this into `SliderHandle`. 
thoughts on `SliderHandleType.HIDDEN` or `SliderInteractionKind.NONE`?

Reply:
Will explore the possibility, I think it could create certain code weirdnesses but I'm not positive.

Reply:
cool can leave that for next time

Reply:
like, after this round of review. still in 3.0